src/main/java/edu/hm/hafner/analysis/parser/pvsstudio/AnalyzerType.java

```diff
@@ -0,0 +1,148 @@
+package edu.hm.hafner.analysis.parser.pvsstudio;
+
+/**
+ * The AnalyzerType for PVS-Studio static analyzer.
+ *
+ * @author PVS-Studio Team
+ */
+@SuppressWarnings("PMD")
+public enum AnalyzerType {
```

Comment:
Actually this enum should be better a expressed with several static classes that implement the same interface
- `String getMessage` from the parser class
- `Optional<AnalysisType> create(int errorCode)` from the enum
Then we have a real OO style algorithm.

Then we additionally need a static factory method `fromErrorCode` and we are done